src/main/io/osd.c

```diff
@@ -770,10 +770,47 @@ static bool osdDrawSingleElement(uint8_t item)
             STATIC_ASSERT(OSD_FORMAT_MESSAGE_BUFFER_SIZE <= sizeof(buff), osd_warnings_size_exceeds_buffer_size);
 
             const batteryState_e batteryState = getBatteryState();
+            const timeUs_t currentTimeUs = micros();
+
+            static timeUs_t armingDisabledUpdateTimeUs;
```

Comment:
Wouldn't ms make more sense here?

Reply:
Probably makes sense.

Reply:
Sorry, I wasn't very clear here. Are you going to change this now?

Reply:
Sorry, I wasn't sure if you were just agreeing or wanted me to change. I went to make the change to milliseconds and realized it's not so simple.  I could easily change the arming disabled section to use millis(), but to change the beacon countdown would be a lot of changes upstream (arming/disarming sequence, beeper, etc.).  So I could either leave it as it is with both sharing a common `currentTimeUs`, or revert back to them being separate and change the new section to using only `millis()` (leaving the beacon countdown as is).

Reply:
Na, leave it as it is then.